BrainPortal/app/views/resource_usage/_resource_usage_table.html.erb

```diff
@@ -98,8 +99,54 @@
                      :value => (@total.to_f / @resource_usages.total_entries.to_f)
                     )
                   ) %>)
+  </p>
   <% end %>
 </fieldset>
+<fieldset>
+  <legend>
+    Additional filtering
+    <%= show_hide_toggle "(show/hide)", ".filtration", :class => 'task_help_link', :slide_effect => true, :slide_duration => 'fast'  %>
+  </legend>
+
+
+  <%
+    is_hidden = (params[:data] && params[:data][:created_at].present?) ||
+                 params[:deleted_items].present?                       ||
+                (params[:positive_file_delta].present? && @maintype =~ /Userfile/) ||
+                (params[:negative_file_delta].present? && @maintype =~ /Userfile/) ?
+                 "" : "hidden"
+  %>
+
+  <div class="simple filtration <%= is_hidden %>">
```

Comment:
`filtration` doesn't exist, it isn't in the stylesheets and it's only used for the show/hide toggle, so rename that.